code/datums/components/item_equipped_movement_rustle.dm

```diff
@@ -0,0 +1,77 @@
+/datum/component/item_equipped_movement_rustle
+	var/default_rustle_sounds = SFX_SUIT_STEP
```

Comment:
this var just should not exist, components should never be subtyped

Reply:
Done